docs/v3/_category_.json

```diff
@@ -1,5 +1,5 @@
 {
-  "label": "V3 Alpha",
+  "label": "Group Chat Alpha",
```

Comment:
Fantastic - thank you so much for this, @scotboyd!

In looking at the preview, I wonder if the "Signature Changes for XMTP v3" content should be updated in the same way? WDYT?

Reply:
The actual topic goes into detail about V2 vs V3 signatures.  The signatures in V3 are changing even for non-group chats, as of this writing.  E.g.:

> In this table, a V2 user has a V2 identity that has not been migrated to V3.

This happens on login regardless of using group chat.

So I would not want to change the title to "Signature Changes for Group Chat" because it implies it won't happen in V3 if you don't use group chat.

Changing the left nav is a navigation issue, not as much a content issue.  If we want to change content for some reason, I need more information about what those reasons are and what we're trying to prevent.